salt/config/__init__.py

```diff
@@ -43,9 +43,10 @@
 
 try:
     import psutil
-    psutil.virtual_memory
+    if not hasattr(psutil, 'virtual_memory'):
+        raise ImportError('Version of psutil too old.')
```

Comment:
Just a note here: https://hynek.me/articles/hasattr/

hasattr basically would do the same as @guettli did before (which is the recommended way).